lib/modules/auth/login/bloc/login_bloc.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+import 'package:bloc/bloc.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:shop_app/models/login/login_model.dart';
+
+import 'package:shop_app/modules/auth/login/bloc/login_repository.dart';
+import 'login_event.dart';
+import 'login_state.dart';
+
+class LoginBloc extends Bloc<LoginEvent, LoginState> {
+  final BaseLoginRepository loginRepository;
+  IconData suffix = Icons.visibility_outlined;
+  bool isPassword = true;
+
+  LoginBloc(this.loginRepository) : super(LoginInitialState());
+
+  static LoginBloc get(BuildContext context) =>
+      BlocProvider.of<LoginBloc>(context);
+
+  @override
+  Stream<LoginState> mapEventToState(LoginEvent event) async* {
+    if (event is UserLoginEvent) {
```

Comment:
is there a better way to handle this case. 
which means when I send the login model from UI and check the validation return state that is a valid values.
what's the better return state that the login value is correct and fire another event to do login for UI (Bloc Listener).

Reply:
???

Reply:
i think this way is better and simple